hash_to_array.rb

```diff
@@ -0,0 +1,29 @@
+#(3) Write a program that collects 5 keys and 5 values from the user, and stores them in a hash. 
+#Write a function that accepts the hash as optional parameters and prints out an array of keys and an array of values. 
+#Call the function within your program so you know it works. 
+#(Question: Can you find online information on Ruby hash methods that will help with this function?)  
+#Call this program hash_to_array.rb.
+
+def collect_list
+    hash = {}
+
+    5.times do
+    print "What is the item? "
+    item_name = gets.chomp
+
+    print "How much does the item cost? "
+    item_cost = gets.chomp
+
+    hash.store("#{item_name}", "#{item_cost}") 
+
+        hash.each do |item_name, item_cost|
+        print "#{item_name}: #{item_cost}"
+        end
```

Comment:
I'm guessing this was just added for testing and also maybe as a nice add-on. :) Might want to use `puts` instead of `print` (as well as in lines 11 and 14) so it will automatically start a new line and look cleaner when it runs.